playpen/ansible/roles/dev/files/pulp-nightlies.repo

```diff
@@ -0,0 +1,7 @@
+[pulp-nightlies]
+name=Pulp Nightlies $releasever - $basearch
+baseurl=https://repos.fedorapeople.org/repos/pulp/pulp/testing/automation/master/fedora-$releasever/$basearch/
+enabled=1
+gpgcheck=0
+metadata_expire=6h
+skip_if_unavailable=0
```

Comment:
What if we added this to the repo files we host on fedorapeople instead? That way it would be more advertised for community people who might want to help test.